mijeong/models/drinkDao.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-unused-vars */
+import prisma from '../../prisma';
+
+const drinksList = async() => {
+  const drinks = await prisma.$queryRaw(`
+  SELECT drinks.id, drinks.korean_name, drinks.english_name,
+  categories.category_name
+  FROM drinks
+  JOIN categories
+  ON categories.id = drinks.category_id;
+  `);
+
```

Comment:
```suggestion
  const drinks = await prisma.$queryRaw(`
    SELECT
      drinks.id,
      drinks.korean_name,
      drinks.english_name,
      categories.category_name
    FROM drinks
    JOIN categories
    ON categories.id = drinks.category_id;
  `);

```
가독성을 위해 줄 분리하셨군요! 좋습니다 👍 
컬럼이 많을 때는 이런식으로 모두 분리하시는 것도 자주 사용되는 방법입니다.